src/jit/emitxarch.cpp

```diff
@@ -3483,7 +3483,7 @@ void emitter::emitIns_R_I(instruction ins, emitAttr attr, regNumber reg, ssize_t
         sz += emitGetRexPrefixSize(ins);
     }
 
-#ifdef _TARGET_X86_
+#if defined(_TARGET_X86_) && defined(LEGACY_BACKEND)
```

Comment:
It looks like AVX functionality is under FEATURE_AVX_SUPPORT.   Enabling that would enable AVX functionality.. So a separate issue might not be needed.

---
In reply to: [85456732](https://github.com/dotnet/coreclr/pull/7864#discussion_r85456732) [](ancestors = 85456732)